### PR TITLE
Feature/preliminary refactoring

### PR DESCRIPTION
**Pull request by jonas-thiele**
### Reason for this PR:

- Small refactoring in preparation for the upcoming changes (using of volumes, moving parts of the network setup into the cluster). Further refactoring is likely required.

### Change is this PR:

- Separate script into smaller subscripts
- Improve readability of output

### CODE REVIEW

File: scripts/setupOrg1Ca.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source ./util.sh
+
+header "Org1 CA"
+
+# Create deployment for org1 ca
+if (($(kubectl get deployment -l app=rca-org1-root --ignore-not-found -n hlf-production-network | wc -l) < 2)); then
```
Comment (palkra) [inline scripts/setupOrg1Ca.sh:6]: Remove conditions

File: scripts/setupOrg1Ca.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  echo "Creating Org1 CA deployment"
+  kubectl create -f $K8S/org1-ca/org1-ca.yaml -n hlf-production-network
+else
+  echo "Org1 CA deployment already exists"
+fi
+
+# Expose service for org1 ca
+if (($(kubectl get service -l app=rca-org1-root --ignore-not-found -n hlf-production-network | wc -l) < 2)); then
```
Comment (palkra) [inline scripts/setupOrg1Ca.sh:14]: Here too

**Review by palkra — CHANGES_REQUESTED** [review]
It appears as if the $DEBUG flags are not supported anymore. Either remove them or add the parameter in the beginning of each script :)

File: scripts/setupOrg2Ca.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
+source ./util.sh
+
+header "Org2 CA"
+
+# Create deployment for org2 ca
+if (($(kubectl get deployment -l app=rca-org2-root --ignore-not-found -n hlf-production-network | wc -l) < 2)); then
```
[the remark this answers is not in the document]
Comment (jonas-thiele) [reply scripts/setupOrg2Ca.sh:6]: > It appears as if the $DEBUG flags are not supported anymore. Either remove them or add the parameter in the beginning of each script :)

The debug flag still works as intended

[the remark this answers is not in the document]
Comment (palkra) [reply scripts/setupOrg2Ca.sh:6]: Okay, then just ignore this comment :)

[the remark this answers is not in the document]
Comment (jonas-thiele) [reply scripts/setupOrg2Ca.sh]: Checks have been removed :)

**Review by palkra — APPROVED** [review]

**Pull request merged**